### PR TITLE
Calculator 

### PR DESCRIPTION
**Pull request by bharathhjsuryasoft**
* Calculator Exercise was implemented using Java Script.

### CODE REVIEW

File: app.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    font-weight: 600;
+    font-size: 30px;
+    line-height: 35px;
+    letter-spacing: 0.1em;
+
+    color: #000000;
+    background: #FFE4C2;
+}
+
+:root{
```
Comment (SahilDhawan) [inline app.css:15]: why root and not light theme? Please let me know your thoughts here.

Comment (SahilDhawan) [inline app.css:15]: why root and not light theme? Please let me know your thoughts here.

File: app.css [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+}
+
+/* ========================== */
+/*      iPhone13ProMax 1      */
+/* ========================== */
+
+#iPhone13ProMax1{
```
Comment (SahilDhawan) [inline app.css:72]: Last I remember we discussed on name change?

Comment (SahilDhawan) [inline app.css:72]: Last I remember we discussed on name change?

File: app.css [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    --stext-color: #000000;
+}
+
+#icon{
+    float: right;
+    margin-right: 100px;
+    padding-top: 20px;
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
```
Comment (SahilDhawan) [inline app.css:65]: why is this required?

Comment (SahilDhawan) [inline app.css:65]: why is this required?

File: app.css [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+    font-weight: 600;
+    font-size: 60px;
+    line-height: 70px;
+    letter-spacing: 0.1em;
+    
+    color: var(--poutput-color);
+
+}
+
+#keyboard{
```
Comment (SahilDhawan) [inline app.css:122]: Why is this required?

Comment (SahilDhawan) [inline app.css:122]: Why is this required?

File: app.css [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+    height: 594px;
+    padding-top: 50px;
+    padding-left: 12px;
+}
+
+
+.operator, .number {
```
Comment (SahilDhawan) [inline app.css:129]: formatting appears to be off for this file. Can you please check?

Comment (SahilDhawan) [inline app.css:129]: formatting appears to be off for this file. Can you please check?

File: app.css [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+    background: var(--pclear-color);
+    border-radius: 30px;
+    border-style: none;
+
+    font-weight: 150;
+    font-size: 25px;
+    color: var(--ptext-color);
+}
+
+.number, .operator{
```
Comment (SahilDhawan) [inline app.css:156]: why are we repeating ourselves here?

Comment (SahilDhawan) [inline app.css:156]: why are we repeating ourselves here?

File: app.css [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+    background:  var(--pnumber-color);
+    color: var(--ptext-color);
+    border-radius: 50%;
+}
+
+.dot{
+    background:  var(--pnumber-color);
+    color: var(--ptext-color);
+    border-radius: 50%;
+}
```
Comment (SahilDhawan) [inline app.css:193]: we are missing end of file here. Please check.

Comment (SahilDhawan) [inline app.css:193]: we are missing end of file here. Please check.

File: app.js [hunk 8]
```diff
@@ -0,0 +1,78 @@
+function getHistory(){
```
Comment (SahilDhawan) [inline app.js:1]: Feedback not addressed?

Comment (SahilDhawan) [inline app.js:1]: Feedback not addressed?

Comment (SahilDhawan) [reply app.js:1]: Stopping the review here since feedback is not addressed.

Comment (SahilDhawan) [reply app.js:1]: Stopping the review here since feedback is not addressed.

**Review by SahilDhawan — CHANGES_REQUESTED** [review]

**Review by SahilDhawan — CHANGES_REQUESTED** [review]

File: app.css (hunk 1 above)
Comment (bharathhjsuryasoft) [reply app.css:15]: :root can be useful for declaring global CSS variables and the :root CSS pseudo-class matches the root element of a tree representing the document also if I use the light theme as a class instead of :root I have to rewrite a conditions to specify colours and write a separate function to switch  Theme So I thought make use of :root set a default theme as white if any other theme matches from device then switch to it would be better here !

File: app.css (hunk 2 above)
Comment (bharathhjsuryasoft) [reply app.css:72]: Container name would be better as in our situation I have changed the code I have to update it here and I will update soon with rest of all changes that you have asked me

File: app.css (hunk 3 above)
Comment (bharathhjsuryasoft) [reply app.css:65]: `#icon {
  float: right;
  margin-right: 50px;
  padding-top: 20px;
  width: 30px;
  height: 30px;
  cursor: pointer;
}`
This part of css is included to icon that provides a toggle option for user to switch theme if it is not needed we can remove it

File: app.js (hunk 8 above)
Comment (bharathhjsuryasoft) [reply app.js:1]: I have changed the Names and I will update it soon !

File: app.css (hunk 4 above)
Comment (bharathhjsuryasoft) [reply app.css:122]: `#keyboard {
  height: 594px;
  padding-top: 50px;
  padding-left: 12px;
}` This is a container for all the buttons in calculator it is required to define outline of the container for buttons.

File: app.css (hunk 5 above)
Comment (bharathhjsuryasoft) [reply app.css:129]: Yeah Sahil I have looked it up and I have taken care of format of code in css

File: app.css (hunk 6 above)
Comment (bharathhjsuryasoft) [reply app.css:156]: I have removed them and placed in above class style itself rectified them now

File: app.css (hunk 7 above)
Comment (bharathhjsuryasoft) [reply app.css:193]: That's the End of CSS file Sahil
